.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: feature-request
+assignees: ''
+
+---
+
+### Is this related to a problem?
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
nit: wordiness.

Suggestion:
A clear and concise description of the issue, e.g. I'm always frustrated when...